circuits/circomHelperConfig.json

```diff
@@ -1,8 +1,6 @@
 {
-    "circom": "./node_modules/circom/cli.js",
+    "circom": "../../.local/bin/circom",
```

Comment:
I just noticed that `circom` package has been removed. Is there a reason we would want to have people build this locally from now on?

Reply:
Yes, circom v2 is a Rust binary that devs will have to build separately (https://docs.circom.io/getting-started/installation/)